pkgs/by-name/vm/vmctl/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  stdenvNoCC,
+  bash,
+  shellcheck,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "vmctl";
+  version = "v0.99-unstable-2024-05-14";
+
+  src = fetchFromGitHub {
+    owner = "SamsungDS";
+    repo = "vmctl";
+    rev = "5b6b7084b8cba06b474c0e020df965237f2c832c";
+    sha256 = "sha256-yDgaY2RJXBjWkMSQb4JaJzMGLFzowfOGixSRzzv2iIk=";
+  };
+
+  dontBuild = true;
+  doCheck = true;
+
+  nativeCheckInputs = [ shellcheck ];
+
```

Comment:
```suggestion
```

We don't want to fail this package if shellcheck gets updated and adds new checks.

Reply:
Hmm, so we just remove it and do ` dontCheck = true;` ?

Reply:
or just remove `docheck = true;`